Header header
string name
int32 id
int32 current_known_robot_count

int32 robot_unregistered = 0
int32 robot_idle = 1
int32 robot_working = 2
int32 robot_failure = 3
int32 robot_lost = 4

int32 robotState
